packages/block-library/src/image/styles.native.scss

```diff
@@ -1,5 +1,21 @@
 // @format
 
+.badge {
+	position: absolute;
+	top: 10px;
+	left: 10px;
+	background-color: rgba(85, 85, 85, 0.7);
+	border-radius: 6px;
+	padding: 3px 6px;
+}
+
+.badgeText {
+	text-transform: uppercase;
+	color: #fff;
+	font-size: 14;
+	font-weight: bold;
```

Comment:
I think this should be `600` here

Reply:
Another good catch. However, with the updated spec, I think the actual value is 500. Updating...